io.asgardio.tomcat.oidc.sample/src/main/webapp/home.jsp

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this required?

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-oidc-agent/pull/7/commits/bf4f7fa7367a16bd78c9cbabc954df5fb7451e18